packages/riipen-ui/src/components/Link.jsx

```diff
@@ -1,148 +1,175 @@
-
-  render() {
-    const {
-      children,
-      classes,
-      color,
-      component: Component,
-      underline,
-      ...other
-    } = this.props;
-
-    const theme = this.context;
-
-    const className = clsx(color, classes);
-
-    return (
-      <React.Fragment>
-        <Component className={className} {...other}>
-          {children}
-        </Component>
-        <style jsx>{`
-          a {
-            cursor: pointer;
-            font-family: ${theme.typography.fontFamily};
-            text-decoration: ${underline === "always" ? "underline" : "none"};
-            transition: ${theme.transitions.standard} all;
-          }
-          a:hover {
-            text-decoration: ${underline === "hover" ? "underline" : "none"};
-          }
-
-          .inherit {
-            color: inherit;
-          }
-
-          .primary {
-            color: ${theme.palette.primary.main};
-          }
-          .primary:active,
-          .primary:focus,
-          .primary:hover {
-            color: ${theme.palette.primary.dark};
-          }
-
-          .secondary {
-            color: ${theme.palette.secondary.main};
-          }
-          .secondary:active,
-          .secondary:focus,
-          .secondary:hover {
-            color: ${theme.palette.secondary.dark};
-          }
-
-          .tertiary {
-            color: ${theme.palette.tertiary.main};
-          }
-          .tertiary:active,
-          .tertiary:focus,
-          .tertiary:hover {
-            color: ${theme.palette.tertiary.dark};
-          }
-
-          .positive {
-            color: ${theme.palette.positive.main};
-          }
-          .positive:active,
-          .positive:focus,
-          .positive:hover {
-            color: ${theme.palette.positive.dark};
-          }
-
-          .negative {
-            color: ${theme.palette.negative.main};
-          }
-          .negative:active,
-          .negative:focus,
-          .negative:hover {
-            color: ${theme.palette.negative.dark};
-          }
-
-          .white {
-            color: ${theme.palette.common.white};
-          }
-          .white:active,
-          .white:focus,
-          .white:hover {
-            color: ${theme.palette.grey[200]};
-          }
-        `}</style>
-      </React.Fragment>
-    );
-  }
-}
-
-export default withClasses(Link);
+  const className = clsx(color, focusVisible ? "focusVisible" : null, classes);
+
+  return (
+    <React.Fragment>
+      <Component
+        ref={ref}
+        className={className}
+        onFocus={handleFocus}
+        onBlur={handleBlur}
+        {...other}
+      >
+        {children}
+      </Component>
+      <style jsx>{`
+        a {
+          cursor: pointer;
+          font-family: ${theme.typography.fontFamily};
+          text-decoration: ${underline === "always" ? "underline" : "none"};
+          transition: ${theme.transitions.standard} all;
+        }
+
+        a:focus {
+          outline: 0;
+        }
```

Comment:
/ will `.focusVisible` be more specific than `:focus` here?

Reply:
classes and psuedo-classes have the same specificity level, so as a.focusVisible is declared below a:focus in this file it will be

long story short: yes